contribute/design/README.md

```diff
@@ -1,29 +1,58 @@
 
-OpenEBS, being container native, seamlessely integrates easily into the container orchestartors via OpenEBS Maya and OpenEBS Volume containers. 
+OpenEBS Storage Controller functionality comprises of serveral micro-services (or containers) that can be classified into two broad categories: OpenEBS Data Plane - that serves the data to the applications and OpenEBS Control Plane - that manages the OpenEBS Volume Containers. If you notice this classification, closely resembling that of a typical Container Orchestrator, it is because OpenEBS Volumes are delivered through containers and these containers are better managed by Container Orchestrators! So OpenEBS Control Plane services work in conjunction with the CO - schedulers, apiserver, etc., 
+ 
+## OpenEBS Volume Container (aka jiva, aka data plane)
 
-## OpenEBS Volume Containers (aka jiva)
+*openebs/jiva* containers are at the heart of the OpenEBS Volume's storage functionality. OpenEBS Volumes are fully isolated user space storage engines that present the block storage via iSCSI and persist the data onto the storage attached to Nodes (aka Docker Hosts). The storage attached to the docker hosts can either be directly attached disks or cloud volumes (like EBS, GPD, etc.,) depending on where the Container Cluster is deployed. 
 
-*openebs/jiva* containers are the heart of the OpenEBS Volume's storage functionality. OpenEBS Volumes provide persistent storage for containers, with resliency against system failures via a distrubuted architecture, faster access to the storage, snapshot and backup capabilities. In addition, provides mechanism for monitoring the usage and enforcing QoS policies. 
+![OpenEBS Data Plane](../../documentation/source/_static/OpenEBS-Data-Plane.png)
 
-OpenEBS Volume (or Jiva) Containes are scheduled onto the container hosts using Orchestration Engines. Jiva containers consume the disk storage from the local host or remote storage using volume plugins (like k8s persistent volumes). OpenEBS Volumes are described in yaml files, just like your application pods, are deployed through objects like Pods, Deployments, Services, StatefulSets etc., 
+OpenEBS Volumes provide persistent storage for containers, with resliency against system failures, faster access to the storage, snapshot and backup capabilities. In addition, provides mechanism for monitoring the usage and enforcing QoS policies. 
 
-Further details on the OpenEBS Volume Container - jiva can be found under the following repo - [openeb/jiva](https://github.com/openebs/jiva)
+The Disks where the data is peristed are called as *Storage Backends*, which can be either host directories, attached block devices or remote disks. Each OpenEBS Volume comprises of an iSCSI Target Container (represented as openebs-vol1 in the above diagram) and one or more Replica Containers (openebs-vol1-R1 and openebs-vol1-R2).
```

Comment:
The Disks where data is persisted
peristed = persisted
which can be either = which can either be 
devices = devices,